app/metrics.yaml

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+fxr_metrics:
+  distribution_name:
+    type: string
+    lifetime: application
+    description: >
+      The distribution name of this application. We use this field to recognize Firefox Reality
+      is distributed to which channels, such as FirefoxReality_wavevr, FirefoxReality_oculusvr, FirefoxReality_googlevr, etc.
+    send_in_pings:
+      - metrics
```

Comment:
This would only really send this metric in the `metrics` ping. Is this the intent? Do you want this metric to be in other pings as well?

Reply:
You're right, we need this for baseline pings either.